apps/emqx_conf/etc/emqx_conf.conf

```diff
@@ -1,5 +1,11 @@
-## NOTE: The configurations in this file will be overridden by
-## `<path-to-emqx-installation>/data/emqx_overrides.conf`
+## NOTE:
+## Configs in this file might be overridden by:
+## 1. Environment variables start with 'EMQX_' prefix.
```

Comment:
nit: Insert 'which' or 'that' in front of 'start' so it reads 'Environment variables that start...'